### ISSUE TITLE
Label QR Code only contains IP address

### ISSUE THREAD

**Issue by Bubu5485** [Part-DB/Part-DB-server#721]
Dear all,

I have a problem when generating a Label in the database.

No matter what I choose and what I put in for data I only get the IP address after reading the QR Code.

I tried in Twig mode and Placeholders mode no change.

Is there a setting which I need to turn on or the system works like that?

Thank you very much for the help.

**Comment by anthology** [comment]
Hi
You generate QR code on the menu tools/label profiles or on a component?
When i test a label profiles, i have juste the IP or my domaine but the complete URL on component label generation

**Comment by Bubu5485** [comment]
Hi,

I tried on both and I get the same results. Is there a work around to get something else on the label?

**Comment by anthology** [comment]
Yes, you Can add component id, storage and many more Le 4 oct. 2024 19:58, Bubu5485 ***@***.***> a écrit :
Hi,
I tried on both and I get the same results. Is there a work around to get something else on the label?

**Comment by Bubu5485** [comment]
I only get in the result is the server IP when I read the QR Code. :(

**Comment by jbtronics** [comment]
Do you access Part-DB via an IP address? Or via a reverse proxy?
If so have you checked, that the reverse proxy is correctly configured (especially on the part-db side with `TRUSTED_PROXIES`?)

The QR codes should contain a URL, where the host part is based on the one which you use to access the Part-DB website. When the QR code URL is odd, then there is most likely an issue with the webserver or reverse proxy configuration, which makes it impossible to resolve the correct URL.

**Comment by anthology** [comment]
Can you post a qrcode for test?Your qrcode reader is good ?Le 4 oct. 2024 20:57, Bubu5485 ***@***.***> a écrit :
I only get in the result is the server IP when I read the QR Code. :(

**Comment by Bubu5485** [comment]
> Do you access Part-DB via an IP address? Or via a reverse proxy?
> If so have you checked, that the reverse proxy is correctly configured (especially on the part-db side with `TRUSTED_PROXIES`?)
> 
> The QR codes should contain a URL, where the host part is based on the one which you use to access the Part-DB website. When the QR code URL is odd, then there is most likely an issue with the webserver or reverse proxy configuration, which makes it impossible to resolve the correct URL.

I access Partdb via IP.

**Comment by Bubu5485** [comment]
I will post it on Monday. Im using my phones camera to read the qrcode.

**Comment by Bubu5485** [comment]
> Can you post a qrcode for test? Your qrcode reader is good ?
[qrcode.pdf](https://github.com/user-attachments/files/17274087/qrcode.pdf)

There you go.

**Comment by Bubu5485** [comment]
> Can you post a qrcode for test? Your qrcode reader is good ?

Hi,

Just tested the QR Code and it takes me to the part so its working perfectly.

Thanks for the help.
